### PR TITLE
organize build/ci/test scripts

### PR DESCRIPTION
**Pull request by fabiokung**
... and share some commonly used gradle toggles during local development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ scheduling containers on agents.
 
 ## Building and Testing
 
+For reference check some of the scripts commonly used by commiters in the `scripts/dev/` folder.
```
Comment (corindwyer) [inline README.md]: `For reference check` -> `For reference, check`

File: scripts/ci/travis/build-release [hunk 2]
```diff
@@ -24,3 +24,4 @@ else
   echo -e 'WARN: Should not be here => Branch ['$TRAVIS_BRANCH']  Tag ['$TRAVIS_TAG']  Pull Request ['$TRAVIS_PULL_REQUEST']'
   ./gradlew build --stacktrace
 fi
+
```
Comment (corindwyer) [inline scripts/ci/travis/build-release]: extra line

File: scripts/ci/travis/install [hunk 3]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-# This script will build the project.
```
Comment (corindwyer) [inline scripts/ci/travis/install]: can you add a one line comment to each of the scripts to indicate what it is and why it should be run?

File: scripts/ci/travis/build-release (hunk 2 above)
Comment (fabiokung) [reply scripts/ci/travis/build-release]: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

tl;dr: POSIX, so why not?

Comment (fabiokung) [reply scripts/ci/travis/build-release]: ah, my bad. My editor config confused me, we don't need 2 extra blank lines at the end.
